.github/workflows/test.yaml

```diff
@@ -26,6 +26,24 @@ jobs:
         name: tmppilot.tar.gz
         path: tmppilot.tar.gz
 
+  push:
+    name: push
+    runs-on: ubuntu-16.04
+    needs: build
+    if: github.ref == 'master' && github.event_name != 'pull_request'
```

Comment:
This should also check `github.repository == 'commaai/openpilot'` so it doesn't run on forks

Reply:
I'll add it in case someone turns on actions on their repo. By default they won't run though.